bin/hwinj/pycbc_generate_hwinj

```diff
@@ -141,6 +141,10 @@ parser.add_argument('--mass2', type=float, required=True,
                     help='Second mass of the binary in solar masses.')
 parser.add_argument('--inclination', type=float, required=True,
                     help='Inclination of the binary in radians.')
+parser.add_argument('--coa-phase', type=float, default=0.0,
+                    help='Reference orbital phase parameter in radians. '
```

Comment:
Is this *actually* orbital phase? Or is it in gravitational wave phase? If the former, we might want to a bit more emphatic, so people are aware of the contradiction in terms.